Mathlib/Analysis/Complex/RealDeriv.lean

```diff
@@ -170,3 +170,21 @@ theorem conformalAt_iff_differentiableAt_or_differentiableAt_comp_conj {f : ℂ
   simp [fderiv_comp _ h_diff conjCLE.differentiableAt, this, conj_conj]
 
 end Conformality
+
+section BigO
+
+namespace Complex
+
+open Topology
+
+lemma isBigO_comp_ofReal {f g : ℂ → ℂ} {x : ℝ} (h : f =O[𝓝 (x : ℂ)] g) :
+    (fun y : ℝ ↦ f y) =O[𝓝 x] (fun y : ℝ ↦ g y) :=
+  h.comp_tendsto <| continuous_ofReal.tendsto x
+
+lemma isBigO_comp_ofReal_nhds_ne {f g : ℂ → ℂ} {x : ℝ} (h : f =O[𝓝[≠] (x : ℂ)] g) :
+    (fun y : ℝ ↦ f y) =O[𝓝[≠] x] (fun y : ℝ ↦ g y) :=
+  h.comp_tendsto <| ((hasDerivAt_id (x : ℂ)).comp_ofReal).tendsto_punctured_nhds one_ne_zero
```

Comment:
There is no need for derivatives here, and these don't belong in this file as a result.
```suggestion
  h.comp_tendsto <| continuous_ofReal.continuousWithinAt.tendsto_nhdsWithin fun _ _ ↦ by simp_all
```

Reply:
OK; moved to `Analysis.Complex.Asymptotics`.